README.md

```diff
@@ -1,168 +1,297 @@
 | --- | --- | --- | --- |
 
-The MSAL library for Android gives your app the ability to use the [Microsoft Cloud](https://cloud.microsoft.com) by supporting [Microsoft Azure Active Directory](https://azure.microsoft.com/en-us/services/active-directory/) and [Microsoft Accounts](https://account.microsoft.com) in a converged experience using industry standard OAuth2 and OpenID Connect. The library also supports [Azure AD B2C](https://azure.microsoft.com/services/active-directory-b2c/).
+The MSAL library for Android gives your app the ability to use the [Microsoft Cloud](https://cloud.microsoft.com) by supporting [Microsoft Azure Active Directory](https://azure.microsoft.com/services/active-directory/) and [Microsoft accounts](https://account.microsoft.com) in a converged experience using industry standard OAuth2 and OpenID Connect. The library also supports [Azure AD B2C](https://azure.microsoft.com/services/active-directory-b2c/).
 
 [![Version Badge](https://img.shields.io/maven-central/v/com.microsoft.identity.client/msal.svg)](http://repo1.maven.org/maven2/com/microsoft/identity/client/msal/)
 [![Build Status](https://travis-ci.org/AzureAD/microsoft-authentication-library-for-android.svg?branch=master)](https://travis-ci.org/AzureAD/microsoft-authentication-library-for-android)
 
-## Important Note about the MSAL Preview
 
-This library can be used in a production environment. We provide the same production level support for this library as we do our current production libraries. During the preview we may make changes to the API, internal cache format, and other mechanisms of this library, which you will be required to take along with bug fixes or feature improvements. This may impact your application. For instance, a change to the cache format may impact your users, such as requiring them to sign in again. An API change may require you to update your code. When we provide the General Availability (GA) release we will require you to update to the GA version within 6 months. Applications written using a preview version of library may no longer work.
+## Introduction
 
-## Example
+### What's new?
 
-```Java
-    // Instantiates MSAL Public Client App
-    PublicClientApplication myApp = new PublicClientApplication(
-                    this.getApplicationContext(),
-                    R.raw.auth_config);
+MSAL Android is now generally available with MSAL 1.0!
 
-    // Acquires a token from AzureAD 
-    myApp.acquireToken(this, SCOPES, getAuthInteractiveCallback());
+- Supported Authorities
+  - Microsoft [identity platform](https://docs.microsoft.com/azure/active-directory/develop/) (also known as the Azure Active Directory v2 Endpoint)
+  - [Azure Active Directory B2C](https://docs.microsoft.com/azure/active-directory-b2c/)
+    - [Integrate with B2C](https://docs.microsoft.com/azure/active-directory/develop/msal-android-b2c)
```

Comment:
This link does not work

Reply:
nit: indentation seems off for the last bullet.